CHANGELOG.mdx

```diff
@@ -5,6 +5,12 @@ route: /changelog
 
 # Changelog
 
+#### [v. 2.6.7]
+
+- Added single item carousel property/styling. Adjusted button link styling.
+
+_Craig Walker_
```

Comment:
nice, I love taking credit for things ;)

Reply:
HAHA incredible work!